extensions/skyportal/skyportal/handlers/api/alert.py

```diff
@@ -107,49 +118,26 @@ def get_user_streams(self):
 
         return streams
 
-    # async def query_kowalski(self, query: dict, timeout=7):
-    #     base_url = f"{self.cfg['app.kowalski.protocol']}://" \
-    #                f"{self.cfg['app.kowalski.host']}:{self.cfg['app.kowalski.port']}"
-    #     headers = {"Authorization": f"Bearer {self.cfg['app.kowalski.token']}"}
-    #
-    #     resp = await c.fetch(
-    #         os.path.join(base_url, 'api/queries'),
-    #         method='POST',
-    #         body=tornado.escape.json_encode(query),
-    #         headers=headers,
-    #         request_timeout=timeout
-    #     )
-    #
-    #     return resp
-
-    def query_kowalski(self, query: dict, timeout=10):
-        base_url = (
-            f"{self.cfg['app.kowalski.protocol']}://"
-            f"{self.cfg['app.kowalski.host']}:{self.cfg['app.kowalski.port']}"
-        )
-        headers = {"Authorization": f"Bearer {self.cfg['app.kowalski.token']}"}
-
-        resp = requests_session.post(
-            os.path.join(base_url, "api/queries"),
-            json=query,
-            headers=headers,
-            timeout=timeout,
-        )
-
-        return resp
-
     @auth_or_token
-    async def get(self, objectId: str = None):
+    async def get(self, object_id: str = None):
         """
         ---
         single:
-          description: Retrieve a ZTF objectId from Kowalski
+          description: Retrieve an object from Kowalski by object_id
```

Comment:
```suggestion
          description: Retrieve an object from Kowalski by objectId
```
I think it might be better to keep this in camel case since the parameter name will stay that way, to avoid confusion

Reply:
Fixed.